packages/main/src/themes/Dialog.css

```diff
@@ -1,51 +1,32 @@
-.ui5-dialog-root-parent {
+:host {
+	background: transparent;
+	display: inline;
+	min-height: auto;
+	min-width: auto;
+	box-shadow: none;
 	position: fixed;
-	left: 0;
-	right: 0;
-	top: 0;
-	bottom: 0;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	overflow: hidden;
+  top: 50%;
+  left: 50%;
+  /* bring your own prefixes */
+  transform: translate(-50%, -50%);
```

Comment:
We need to center dialog in the middle of the screen. `left: 50%` moves it 50% from the left, but we need 50% - half of its width. As this could not be really achieve by just using CSS, we are using transformation to flip the dialog and center it correctly.